.github/workflows/ci.yml

```diff
@@ -78,9 +78,6 @@ jobs:
       - name: Install dependencies
         run: pnpm install
 
-      - name: Run build
```

Comment:
Not sure why we were running the build for Unit and Lint workflows. It is run in the build workflow